web/src/lib/components/faces-page/unassigned-faces-side-panel.svelte

```diff
@@ -0,0 +1,204 @@
+    </div>
+  </div>
+  {#if unassignedFaces.length > 0 && unassignedFaces.length > Object.keys(selectedPersonToAdd).length}
+    <div class="px-4 py-4 text-sm">
+      <p>Faces removed</p>
+      <div class="mt-4 flex flex-wrap gap-2">
+        {#each unassignedFaces as face, index (face.id)}
+          {#if !selectedPersonToAdd[face.id]}
+            <div class="relative z-[20001] h-[115px] w-[95px]">
+              <button
+                tabindex={index}
+                class="absolute left-0 top-0 h-[90px] w-[90px] cursor-default"
+                on:focus={() => (face ? ($boundingBoxesArray = [face]) : '')}
+                on:mouseover={() => (face ? ($boundingBoxesArray = [face]) : '')}
+                on:mouseleave={() => ($boundingBoxesArray = [])}
+              >
+                <ImageThumbnail
+                  curve
+                  shadow
+                  url={face.customThumbnail}
+                  title="Available face"
+                  altText="Available face"
+                  widthStyle="90px"
+                  heightStyle="90px"
+                  thumbhash={null}
+                />
+                <div class="absolute -right-[8px] -top-[8px] h-[20px] w-[20px]">
+                  <CircleIconButton
+                    color="blue"
+                    icon={mdiMinus}
+                    title="Reset"
+                    size="20"
+                    buttonSize="20"
+                    padding="[1px]"
+                    disableHover
+                    class="absolute  left-1/2 top-1/2 translate-x-[-50%] translate-y-[-50%]"
+                    on:click={() => handleSelectedFace(face)}
+                  />
+                </div>
+              </button>
+            </div>
+          {/if}
+        {/each}
+      </div>
+    </div>
+  {:else}
+    <div class="flex items-center justify-center">
+      <div class="grid place-items-center">
+        <Icon path={mdiAccountOff} size="3.5em" />
+        <p class="mt-5 font-medium">No faces removed</p>
+      </div>
+    </div>
+  {/if}
+  {#if Object.keys(selectedFaceToRemove).length > 0}
+    <div class="px-4 py-4 text-sm">
+      <div class="flex items-center justify-between">
+        <p>Faces to be removed</p>
+        <button
+          class="justify-self-end rounded-lg p-2 hover:bg-immich-dark-primary hover:dark:bg-immich-dark-primary/50"
+          on:click={handleRemoveAllFaces}
+          title="Reset"
+        >
+          <div>
+            <Icon path={mdiClose} />
+          </div>
+        </button>
+      </div>
+      <div class="mt-4 flex flex-wrap gap-2">
+        {#each Object.entries(selectedFaceToRemove) as [id, face], index}
+          <div class="relative z-[20001] h-[115px] w-[95px]">
+            <button
+              tabindex={index}
+              class="absolute left-0 top-0 h-[90px] w-[90px] cursor-default"
+              on:focus={() => (face ? ($boundingBoxesArray = [face]) : '')}
```

Comment:
I don't understand this. If `face` is set you assign some variable and otherwise you return an empty string? Why return that string?

Reply:
fixed